packages/odyssey-react-mui/src/Tag.tsx

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Chip, ChipProps } from ".";
+import { memo } from "react";
+
+export interface TagProps extends Omit<ChipProps, "clickable" | "disabled"> {
+  isClickable?: boolean;
+  isDisabled?: boolean;
+}
+
+const Tag = ({ isClickable, isDisabled, label, ...rest }: TagProps) => (
+  <Chip label={label} clickable={isClickable} disabled={isDisabled} {...rest} />
+);
```

Comment:
I'd like to avoid using `...rest` on Odyssey component going forward. If there's a prop we allow, we should allow it. The `TagProps` type doesn't even define `...rest`, so it doesn't make sense to include it as someone would have to `ts-ignore`.

Reply:
**OPINION**

I would also rename `onDelete` to `onRemove`.

- "Delete" is what you do to data like deleting it from a database.
- "Remove" is something you do to UI like removing a row from a table.

When I think "delete", I think "gone for good". But you can always bring back that tag.

What do you think?